README.md

```diff
@@ -1,27 +1,56 @@
-# Zenodo spam classifiers
+## Zenodo spam classifiers
 
 Spam classification machine learning models for Zenodo records and communities.
 
 ## Usage
 
-First of all, create a virtualenv, install the depencencies, and run the Jupyter notebook server:
+First of all, create a virtual environment (the make script will install the required dependencies in it):
 
 ```bash
-# Create a virtual environment
-   mkvirtualenv --python python3.9 zenodo-classifier
-   (zenodo-classifier) pip install -e .
-   
-# This will also open Jupyter notebook in your browser
-   (zenodo-classifier) jupyter notebook
+   mkvirtualenv --python python3.10 zenodo-classifier # Create the virtual environment
 ```
 
-To re-train the model:
+To train/re-train the model:
 
-1. Go to Zenodo Open Metadata record at <https://doi.org/10.5281/zenodo.787062> to acces all dataset versions.
```

Comment:
We can keep the reference to the dataset DOI, to make it easier to get hold of the (test) data for training.